src/modules/wallet/components/overview/overview.js

```diff
@@ -104,7 +104,7 @@ const Overview = ({ isWalletRoute, history }) => {
           copy
           size={50}
           address={authData?.meta?.address}
-          accountName={searchAddress || validator.name ? validator.name : name}
```

Comment:
You have to leave this as it was, when we search for a validator, in the wallet page it should show the searched validator name.

Actual

![Screenshot 2023-07-31 at 10 54 39 PM](https://github.com/LiskHQ/lisk-desktop/assets/6449871/2614d9cd-12da-4889-87a0-9de54fb1f727)


Expected

![Screenshot 2023-07-31 at 10 54 21 PM](https://github.com/LiskHQ/lisk-desktop/assets/6449871/56e81333-7842-4924-9301-3f07649776e7)

Reply:
Hmm, I dont think we can leave as it was? Whole ticket was about not showing the validator name for current account. 

Made some modifications here that should account for the state in this conversation, please check:

https://github.com/LiskHQ/lisk-desktop/pull/5213/commits/4c8c977962cc8d14f4c53adbe3dac8088996d89e